modules/core/src/test/scala/doobie/util/FragmentsSuite.scala

```diff
@@ -37,6 +38,30 @@ class FragmentsSuite extends munit.FunSuite {
     assertEquals(values(NonEmptyList.of((1, true), (2, false))).query[Unit].sql, "VALUES (?,?) , (?,?) ")
   }
 
+  test("updateSetOpt for three column") {
+    assertEquals(
+      updateSetOpt(Fragment.const("Foo"), sqlKv, List.empty[Fragment]).map(_.query[Unit].sql),
+      Some("UPDATE Foo SET a = ?, b = ?, c = ? "))
```

Comment:
Here, we currently have an empty space, if the `WHERE` clause is empty. Let me know, if this should be handled differently.

Reply:
The spaces are fine :)